### PR TITLE
[cli][client] (Major) Remove legacy Now CLI 1.0 code

### PR DESCRIPTION
**Pull request by MaxLeiter**
It was easier to make a new PR than re-base the old ones:

https://github.com/zeit/now/pull/4017
https://github.com/vercel/vercel/pull/4602

Story: https://app.clubhouse.io/vercel/story/425/remove-all-the-zeit-now-1-0-code-from-vercel-cli

### CODE REVIEW

File: packages/now-cli/test/unit.js [hunk 1]
```diff
@@ -2,389 +2,21 @@ import { basename, join, sep } from 'path';
...
-
-  t.is(files.length, 4);
-
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-  t.is(base(files[3]), `${path}/package.json`);
-});
-
-test('should observe .vercelignore file', async t => {
```
Comment (styfle) [inline packages/now-cli/test/unit.js:239]: We need to keep tests for `getStaticFiles()` which is still used for v2

Comment (MaxLeiter) [reply packages/now-cli/test/unit.js:239]: I re-added the vercelignore test

File: packages/now-cli/src/commands/alias/set.ts [hunk 2]
```diff
@@ -133,62 +96,79 @@ export default async function set(
...
-        )} to ${target}`
+    if (!deployment) {
+      output.error(
+        `Couldn't find a deployment to alias. Please provide one as an argument.`
       );
-      const pathAlias = await upsertPathAlias(
+      return 1;
+    }
+
+    // Find the targets to perform the alias	  const [deploymentIdOrHost, aliasTarget] = args;
```
Comment (styfle) [inline packages/now-cli/src/commands/alias/set.ts]: Is the `const [deploymentIdOrHost, aliasTarget] = args;` supposed to be commented out?

File: packages/now-cli/src/commands/alias/set.ts [hunk 3]
```diff
@@ -133,62 +96,79 @@ export default async function set(
...
+    const targets = getTargetsForAlias(args, localConfig);
+
+    if (targets instanceof Error) {
+      output.error(targets.message);
```
Comment (styfle) [inline packages/now-cli/src/commands/alias/set.ts]:
```suggestion
      output.prettyError(targets);
```

File: packages/now-cli/src/commands/alias/set.ts [hunk 4]
```diff
@@ -225,36 +205,31 @@ export default async function set(
     return 1;
   }
 
-  // Assign the alias for each of the targets in the array
-  for (const target of targets) {
-    output.log(`Assigning alias ${target} to deployment ${deployment.url}`);
+  output.log(`Assigning alias ${aliasTarget} to deployment ${deployment.url}`);
```
Comment (styfle) [inline packages/now-cli/src/commands/alias/set.ts:208]: I see this message `Assigning alias` twice, once here and once on line 139

Is that correct?

File: packages/now-cli/src/commands/deploy/index.js [hunk 5]
```diff
@@ -113,88 +85,5 @@ export default async ctx => {
...
-    const file = highlight(localConfig[fileNameSymbol]);
-    const prop = code('version');
-
-    if (version) {
-      if (typeof version === 'number') {
-        if (version !== 1 && version !== 2) {
-          const first = code(1);
-          const second = code(2);
-
-          output.error(
```
Comment (styfle) [inline packages/now-cli/src/commands/deploy/index.js:127]: We might want to keep validation to make sure `version` is either `undefined` or `2`. Because in the future when we add version=3, then we would want the old CLI to fail fast and not attempt to read the rest of the config. Maybe that validation needs to happen when reading the `vercel.json` file instead of deploying.

File: packages/now-cli/src/commands/inspect.js [hunk 6]
```diff
@@ -130,35 +129,12 @@ export default async function main(ctx) {
...
-  } = deployment;
-
-  const isBuilds = version === 2;
-  const buildsUrl = `/v1/now/deployments/${finalId}/builds`;
-
-  const [scale, events, { builds }] = await Promise.all([
-    caught(
-      now.fetch(`/v3/now/deployments/${encodeURIComponent(finalId)}/instances`)
-    ),
-    type === STATIC
```
Comment (styfle) [inline packages/now-cli/src/commands/inspect.js:155]: @TooTallNate Are we still allowing STATIC deployments? Do we need to keep this?

File: packages/now-cli/src/commands/inspect.js [hunk 7]
```diff
@@ -214,95 +178,7 @@ export default async function main(ctx) {
     print(`\n\n`);
   }
 
-  if (limits) {
```
Comment (styfle) [inline packages/now-cli/src/commands/inspect.js:217]: I think we still have limits for v2, although I'm not sure if these are included.

See the [`functions`](https://vercel.com/docs/configuration#project/functions) property for an example.

File: packages/now-cli/src/commands/inspect.js [hunk 8]
```diff
@@ -314,6 +190,9 @@ function stateString(s) {
     case 'ERROR':
       return chalk.red(s);
 
+    case 'BUILDING':
+      return chalk.grey(s);
+
```
Comment (styfle) [inline packages/now-cli/src/commands/inspect.js]: We're missing `QUEUED` and `CANCELED`.

Here's all the possibilities:

```
  readyState:
    | 'BUILDING'
    | 'ERROR'
    | 'INITIALIZING'
    | 'QUEUED'
    | 'READY'
    | 'CANCELED';
```

Comment (styfle) [reply packages/now-cli/src/commands/inspect.js]: I think the default option on line 200 should be `return chalk.gray(s || 'UNKNOWN');`

File: packages/now-cli/src/commands/inspect.js (hunk 6 above)
Comment (TooTallNate) [reply packages/now-cli/src/commands/inspect.js:155]: There are no longer any "STATIC" type deployments in the system. They were all upgraded to v2 with `@vercel/static` outputs.

File: packages/now-cli/src/commands/list.js [hunk 9]
```diff
@@ -222,51 +218,8 @@ export default async function main(ctx) {
...
-      );
-      if (match !== null && typeof match !== 'undefined') {
-        deployments = Array.of(match);
-      }
-    }
-  }
-
   now.close();
 
-  if (argv['--all']) {
```
Comment (styfle) [inline packages/now-cli/src/commands/list.js:261]: Did you forget to remove `--all` from the arg parsing above?

File: packages/now-cli/src/types.ts [hunk 10]
```diff
@@ -102,13 +100,6 @@ export type Cert = {
...
 };
 
-export type DeploymentScale = {
-  [dc: string]: {
-    min: number;
-    max: number;
-  };
-};
-
 export type NpmDeployment = {
```
Comment (styfle) [inline packages/now-cli/src/types.ts]: I think you can delete `NpmDeployment`

File: packages/now-cli/src/types.ts [hunk 11]
```diff
@@ -119,7 +110,6 @@ export type NpmDeployment = {
   created: number;
   creator: { uid: string };
   sessionAffinity: string;
-  scale: DeploymentScale;
 };
 
 export type StaticDeployment = {
```
Comment (styfle) [inline packages/now-cli/src/types.ts]: @TooTallNate Is `StaticDeployment` still used?

Comment (MaxLeiter) [reply packages/now-cli/src/types.ts]: I think I can merge `NpmDeployment` and `StaticDeployment` into just `Deployment` now? They're only used in this file

Comment (MaxLeiter) [reply packages/now-cli/src/types.ts]: Saw your above comment

File: packages/now-cli/src/commands/alias/set.ts (hunk 4 above)
Comment (MaxLeiter) [reply packages/now-cli/src/commands/alias/set.ts:208]: The first message is for a single argument in `now alias set <argument>`

File: packages/now-cli/src/util/dev/server.ts [hunk 12]
```diff
@@ -550,8 +550,8 @@ export default class DevServer {
       const featHandleMiss = true; // enable for zero config
       const { projectSettings, cleanUrls, trailingSlash } = config;
 
-      const opts = { output: this.output, isBuilds: true };
-      const files = (await getFiles(this.cwd, config, opts)).map(f =>
+      const opts = { output: this.output };
+      const files = (await getFiles(this.cwd, opts)).map(f =>
```
Comment (styfle) [inline packages/now-cli/src/util/dev/server.ts:554]: ~~What happened to the `config` parameter?~~

I see now, getFiles is really staticFiles and it was removed ☺️

Comment (MaxLeiter) [reply packages/now-cli/src/util/dev/server.ts:554]: The config file was being used for accessing `files: []`, which is no longer necessary.

File: packages/now-cli/test/fixtures/unit/big-file/package.json [hunk 13]
```diff
@@ -1,6 +0,0 @@
-{
-  "name": "",
-  "version": "0.0.1",
-  "description": "",
-  "dependencies": {}
-}
```
Comment (styfle) [inline packages/now-cli/test/fixtures/unit/big-file/package.json:6]: Why is this `big-file` test deleted? It doesn't look like v1?

File: packages/now-cli/test/fixtures/unit/dev-nodejs/now.json [hunk 14]
```diff
@@ -1,19 +0,0 @@
-{
-  "version": 2,
```
Comment (styfle) [inline packages/now-cli/test/fixtures/unit/dev-nodejs/now.json:2]: This is a v2 test. Why is it deleted?

File: packages/now-cli/test/helpers/prepare.js [hunk 15]
```diff
@@ -144,52 +53,41 @@ module.exports = async session => {
...
-      'now.json':
-        '{ "builds": [ { "src": "index.js", "use": "@now/node" } ], "version": 2 }',
-      'package.json': `
-{
-  "scripts": {
-    "now-build": "node now-build.js"
-  }
-}
-      `,
-      'now-build.js': `
```
Comment (styfle) [inline packages/now-cli/test/helpers/prepare.js:178]: Why was this file `now-build.js` deleted? Isn't it necessary?

File: packages/now-cli/test/helpers/prepare.js [hunk 16]
```diff
@@ -240,103 +128,42 @@ module.exports = (req, res) => {
...
         name: 'redirects-v2',
         redirects: [{ source: `/(.*)`, destination: 'https://example.com/$1' }],
       }),
     },
     'deploy-with-only-readme-now-json': {
-      'now.json': JSON.stringify({ version: 2 }),
       'README.md': 'readme contents',
     },
     'deploy-with-only-readme-vercel-json': {
-      'vercel.json': JSON.stringify({ version: 2 }),
```
Comment (styfle) [inline packages/now-cli/test/helpers/prepare.js]: Add back vercel.json, see test name

File: packages/now-cli/test/helpers/prepare.js [hunk 17]
```diff
@@ -240,103 +128,42 @@ module.exports = (req, res) => {
...
     },
     'redirects-v2': {
       'now.json': JSON.stringify({
-        version: 2,
         name: 'redirects-v2',
         redirects: [{ source: `/(.*)`, destination: 'https://example.com/$1' }],
       }),
     },
     'deploy-with-only-readme-now-json': {
-      'now.json': JSON.stringify({ version: 2 }),
```
Comment (styfle) [inline packages/now-cli/test/helpers/prepare.js]: Add back now.json, see test name

File: packages/now-cli/test/helpers/prepare.js [hunk 18]
```diff
@@ -144,52 +53,41 @@ module.exports = async session => {
...
+            src: '/(.*)',
+            status: 301,
+            headers: {
+              Location: 'https://google.com',
+            },
+          },
+        ],
+      }),
+    },
+    'build-env': {
```
Comment (styfle) [inline packages/now-cli/test/helpers/prepare.js:68]: ~~It looks like you added a new fixture for `build-env`? Is this used?~~

I see it is a new test here: https://github.com/vercel/vercel/pull/5011/files?diff=split&w=1#diff-6324a33fb2da36e19435c2c6c5368767R1731

File: packages/now-cli/test/helpers/prepare.js [hunk 19]
```diff
@@ -547,34 +378,21 @@ CMD ["node", "index.js"]`,
...
-      for (const name of names) {
-        const file = join(directory, name);
-        const content = needed[name];
-        await mkdirp(dirname(file));
-        await writeFile(file.replace('-builds', ''), content);
-      }
+    for (const [name, content] of Object.entries(needed)) {
+      const file = join(directory, name);
+      await mkdirp(dirname(file));
+      await writeFile(file.replace('-builds', ''), content);
```
Comment (styfle) [inline packages/now-cli/test/helpers/prepare.js]: Do we still need to replace `-builds`? I thought that was strange behavior before.

File: packages/now-cli/test/integration.js [hunk 20]
```diff
@@ -1180,7 +1164,7 @@ test('try to move an invalid domain', async t => {
 test('try to set default without existing payment method', async t => {
   const { stderr, stdout, exitCode } = await execa(
     binaryPath,
-    ['billing', 'set-default', ...defaultArgs],
+    ['billing', 'set-default', '-y', ...defaultArgs],
```
Comment (styfle) [inline packages/now-cli/test/integration.js]: Why was `-y` added?

File: packages/now-cli/test/integration.js [hunk 21]
```diff
@@ -1197,7 +1181,7 @@ test('try to set default without existing payment method', async t => {
 test('try to remove a non-existing payment method', async t => {
   const { stderr, stdout, exitCode } = await execa(
     binaryPath,
-    ['billing', 'rm', 'card_d2j32d9382jr928rd', ...defaultArgs],
+    ['billing', 'rm', 'card_d2j32d9382jr928rd', '-y', ...defaultArgs],
```
Comment (styfle) [inline packages/now-cli/test/integration.js]: Why was `-y` added?

File: packages/now-cli/test/integration.js [hunk 22]
```diff
@@ -1296,47 +1214,18 @@ test('set platform version using `-V` to `2`', async t => {
...
-  console.log(stderr);
-  console.log(stdout);
-  console.log(exitCode);
-
-  t.true(stderr.includes(`Fetched deployment "${context.deployment}"`));
   t.is(exitCode, 0);
 });
 
 test('output logs of a 2.0 deployment without annotate', async t => {
+  const deployment = await createDeploymentWithFixture();
```
Comment (styfle) [inline packages/now-cli/test/integration.js]: Why is this creating a deployment instead of printing the logs of the previous deployment?

File: packages/now-cli/test/integration.js [hunk 23]
```diff
@@ -1915,6 +1728,73 @@ test('deploying more than 1 path should fail', async t => {
...
+      reject: false,
+    }
+  );
+
+  console.log(stderr);
+  console.log(stdout);
+  console.log(exitCode);
+
+  // Ensure the exit code is right
+  t.is(exitCode, 0, `Received:\n"${stderr}"\n"${stdout}"`);
```
Comment (styfle) [inline packages/now-cli/test/integration.js]:
```suggestion
  t.is(exitCode, 0, formatOutput({ stderr, stdout }));
```

File: packages/now-cli/test/integration.js [hunk 24]
```diff
@@ -1915,6 +1728,73 @@ test('deploying more than 1 path should fail', async t => {
...
+      '--confirm',
+    ],
+    {
+      reject: false,
+    }
+  );
+
+  console.log(stderr);
+  console.log(stdout);
+  console.log(exitCode);
```
Comment (styfle) [inline packages/now-cli/test/integration.js]: You can probably remove these logs

File: packages/now-cli/test/integration.js [hunk 25]
```diff
@@ -3111,7 +2974,7 @@ test('whoami with local .vercel scope', async t => {
   await ensureDir(path.join(directory, '.vercel'));
   await fs.writeFile(
     path.join(directory, '.vercel', 'project.json'),
-    JSON.stringify({ orgId: user.uid, projectId: 'xxx' })
+    JSON.stringify({ orgId: user.uid })
```
Comment (styfle) [inline packages/now-cli/test/integration.js]: Why was `projectId` removed?

File: packages/now-cli/test/unit.js [hunk 26]
```diff
@@ -75,170 +37,9 @@ const normalizeWindowsPaths = files => {
...
-
-  t.is(files.length, 3);
-  t.is(base(files[0]), `${path}/a.js`);
-  t.is(base(files[1]), `${path}/b.js`);
-  t.is(base(files[2]), `${path}/build/a/c.js`);
-});
-
-test('discover static files without `now.files`', async t => {
-  const path = 'now-json-static-no-files';
-  let files = await getStaticFiles(fixture(path));
```
Comment (styfle) [inline packages/now-cli/test/unit.js:215]: I think we need this test `now-json-static-no-files`

File: packages/now-client/src/create-deployment.ts [hunk 27]
```diff
@@ -164,16 +132,8 @@ export default function buildCreateDeployment(version: number) {
       debug(`Using provided user agent: ${clientOptions.userAgent}`);
     }
 
-    debug(`Setting platform version to ${version}`);
-    deploymentOptions.version = version;
```
Comment (styfle) [inline packages/now-client/src/create-deployment.ts:168]: I think this might need to be hard coded to `2` 🤔

File: packages/now-client/src/deploy.ts [hunk 28]
```diff
@@ -26,58 +25,59 @@ async function* createDeployment(
...
-        yield { type: 'tip', payload: value };
-      }
+  for (const [name, value] of response.headers.entries()) {
+    if (name.startsWith('x-now-warning-')) {
+      debug('Deployment created with a warning:', value);
+      yield { type: 'warning', payload: value };
     }
 
-    yield { type: 'created', payload: json };
-  } catch (e) {
```
Comment (styfle) [inline packages/now-client/src/deploy.ts:78]: Why was the `catch` removed?

File: packages/now-client/src/deploy.ts [hunk 29]
```diff
@@ -145,41 +144,11 @@ export async function* deploy(
...
-  if (
-    deploymentOptions.version === 1 &&
-    !deploymentOptions.forceNew &&
-    clientOptions.force
-  ) {
-    debug(`Setting 'forceNew' for 1.0 deployment`);
-    deploymentOptions.forceNew = clientOptions.force;
-  }
-
-  if (clientOptions.withCache) {
```
Comment (styfle) [inline packages/now-client/src/deploy.ts:172]: Why remove `withCache` log here?

File: packages/now-cli/test/fixtures/unit/big-file/package.json (hunk 13 above)
Comment (MaxLeiter) [reply packages/now-cli/test/fixtures/unit/big-file/package.json:6]: It's not being used

File: packages/now-cli/test/fixtures/unit/dev-nodejs/now.json (hunk 14 above)
Comment (MaxLeiter) [reply packages/now-cli/test/fixtures/unit/dev-nodejs/now.json:2]: Same as big-file

File: packages/now-client/tests/fixtures/vercelignore-and-nowignore/.vercelignore [hunk 30]
```diff
@@ -1 +0,0 @@
-foo
```
Comment (styfle) [inline packages/now-client/tests/fixtures/vercelignore-and-nowignore/.vercelignore]: We need this `vercelignore-and-nowignore` test fixture

**Review by styfle — COMMENTED** [review]
Looks like the permissions changed fro `fixtures/unit/directory-main/a/index.js → testing`, was that intentional?

**Comment by MaxLeiter** [comment]
> Looks like the permissions changed fro `fixtures/unit/directory-main/a/index.js → testing`, was that intentional?

Just a file created when I was trying to trigger CI. index.js was a blank file I removed so github got confused. It will be removed 👍

File: packages/now-cli/test/helpers/prepare.js (hunk 19 above)
Comment (MaxLeiter) [reply packages/now-cli/test/helpers/prepare.js]: I don't think so. I don't see `-builds` used anywhere.

File: packages/now-cli/src/util/index.js [hunk 31]
```diff
@@ -199,56 +98,10 @@ export default class Now extends EventEmitter {
...
     });
 
-    // We report about files whose sizes are too big
-    let missingVersion = false;
-
-    if (deployment && deployment.warnings) {
-      let sizeExceeded = 0;
-
-      deployment.warnings.forEach(warning => {
-        if (warning.reason === 'size_limit_exceeded') {
```
Comment (styfle) [inline packages/now-cli/src/util/index.js:209]: I think we still need to print `size_limit_exceeded` warnings, right?

File: packages/now-cli/src/commands/inspect.js (hunk 7 above)
Comment (MaxLeiter) [reply packages/now-cli/src/commands/inspect.js:217]: Confirmed on Slack that these are V1 limits (Cloud V2)

Comment (styfle) [reply packages/now-cli/src/commands/inspect.js:217]: Nate confirmed, these limits are v1 only

File: packages/now-client/tests/fixtures/vercelignore-and-nowignore/.vercelignore (hunk 30 above)
Comment (styfle) [reply packages/now-client/tests/fixtures/vercelignore-and-nowignore/.vercelignore]: Why remove `foo`?

**Review by styfle — DISMISSED** [review]
Great job, thanks! 💯

File: packages/now-cli/src/util/credit-cards.js [hunk 32]
```diff
@@ -38,7 +38,7 @@ export default class CreditCards extends Now {
     return true;
   }
 
-  add(card) {
+  async add(card) {
```
Comment (TooTallNate) [inline packages/now-cli/src/util/credit-cards.js:41]: Why was this changed? Should we refactor to remove the `new Promise()` below?

File: packages/now-cli/src/util/deploy/get-deployment-by-id-or-host.ts [hunk 33]
```diff
@@ -8,13 +8,13 @@ import {
...
 
-type APIVersion = 'v5' | 'v10';
+type APIVersion = 'v12';
 
 export default async function getDeploymentByIdOrHost(
   client: Client,
   contextName: string,
   idOrHost: string,
-  apiVersion: APIVersion = 'v5'
+  apiVersion: APIVersion = 'v12'
```
Comment (TooTallNate) [inline packages/now-cli/src/util/deploy/get-deployment-by-id-or-host.ts]: I'd suggest just using `string` type here.

```suggestion
  apiVersion = 'v12'
```

File: packages/now-cli/src/util/credit-cards.js (hunk 32 above)
Comment (MaxLeiter) [reply packages/now-cli/src/util/credit-cards.js:41]: I changed it for uniformity and forgot to remove the promise... Fixed

File: packages/now-cli/src/types.ts [hunk 34]
```diff
@@ -102,59 +100,23 @@ export type Cert = {
...
-  version?: number;
-  created: number;
-  creator: { uid: string };
-  sessionAffinity: string;
-  scale: DeploymentScale;
-};
-
-export type StaticDeployment = {
+export type Deployment = {
   uid: string;
```
Comment (styfle) [inline packages/now-cli/src/types.ts:104]: Does this `uid` need to be `id`?

File: packages/now-cli/src/types.ts [hunk 35]
```diff
@@ -102,59 +100,23 @@ export type Cert = {
...
-  state: 'INITIALIZING' | 'FROZEN' | 'READY' | 'ERROR';
+  state:
+    | 'BUILDING'
+    | 'ERROR'
+    | 'INITIALIZING'
+    | 'QUEUED'
+    | 'READY'
+    | 'CANCELED';
   version?: number;
   created: number;
```
Comment (styfle) [inline packages/now-cli/src/types.ts:116]: Does this `created` need to be `createdAt`?

**Review by TooTallNate — DISMISSED** [review]

File: packages/now-cli/test/integration.js [hunk 36]
```diff
@@ -1215,170 +1186,30 @@ test('try to remove a non-existing payment method', async t => {
...
-
-  console.log(stderr);
-  console.log(stdout);
-  console.log(exitCode);
-
-  t.true(stderr.includes(`Fetched deployment "${context.deployment}"`));
-  t.is(exitCode, 0);
-});
-
-test('output logs of a 2.0 deployment without annotate', async t => {
```
Comment (styfle) [inline packages/now-cli/test/integration.js:1336]: Why was this 2.0 test deleted?

File: packages/now-cli/test/integration.js [hunk 37]
```diff
@@ -1215,170 +1186,30 @@ test('try to remove a non-existing payment method', async t => {
...
-  t.is(exitCode, 0);
-});
-
 /*
  * Disabled 2 tests because these temp users don't have certs
 test('create wildcard alias for deployment', async t => {
-  const hosts = {
-    deployment: context.deployment,
-    alias: `*.${contextName}.now.sh`,
-  };
```
Comment (styfle) [inline packages/now-cli/test/integration.js:1364]: Why modify a test that is commented out?

File: packages/now-cli/test/integration.js (hunk 36 above)
Comment (styfle) [reply packages/now-cli/test/integration.js:1336]: Oh I see it was moved below

File: packages/now-cli/test/integration.js [hunk 38]
```diff
@@ -2238,25 +2065,8 @@ test('fail `now dev` dev script without now.json', async t => {
...
-
-  // It is expected to fail,
-  // since the package.json does not have a start script
-  t.is(exitCode, 1);
-  t.regex(stderr, /migrate-to-vercel/);
-});
-
 test('`vercel rm` removes a deployment', async t => {
-  const directory = fixture('builds');
+  const directory = fixture('now-dev-fail-dev-script');
```
Comment (styfle) [inline packages/now-cli/test/integration.js]: I don't think you can use `now-dev-fail-dev-script` because it didn't create a deployment so there is nothing to delete. Let's try a previous test we expect to be deployed properly.

```suggestion
  const directory = fixture('static-deployment');
```

File: packages/now-cli/src/types.ts [hunk 39]
```diff
@@ -102,59 +100,23 @@ export type Cert = {
...
-  sessionAffinity: string;
-  scale: DeploymentScale;
-};
-
-export type StaticDeployment = {
+export type Deployment = {
   uid: string;
   url: string;
   name: string;
   type: 'STATIC';
```
Comment (styfle) [inline packages/now-cli/src/types.ts]:
```suggestion
  type: 'LAMBDAS';
```

File: packages/now-cli/test/integration.js (hunk 37 above)
Comment (MaxLeiter) [reply packages/now-cli/test/integration.js:1364]: I previously had a helper function that simplified this test along with a few others (that were removed/rolled back). Fixed

**Review by TooTallNate — DISMISSED** [review]

**Review by styfle — APPROVED** [review]
:shipit:

**Review by TooTallNate — APPROVED** [review]
